### DISCUSSION TITLE
Let us customize our homepage

### DISCUSSION THREAD

**Discussion by cfrikke** [community/community#75225] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I think it would be **AMAZING** if I could put a specific repo on my homepage, so as soon as I load up github.com, I could jump *RIGHT* in to the project I've been working on, instead of having to go and find it, I can star it or something and it would be there right away on [my homepage](https://github.com).

**Comment by reactVishnu** [comment]
yes, please accept my answer bud

**Comment by elderguardian** [comment, marked as answer]
Your top repositories are on the right side of the homepage. Just click on them.
If you do not like that you can try to use a plugin to redirect you (like this https://addons.mozilla.org/en-US/firefox/addon/redirector/).

Or you can just create shortcuts on your desktop

**Reply by cfrikke** [reply]
Sure, but I have 2 accounts, so I can't actually put the repo on the right side due to the fact that it's not my repo, just one I work on

**Reply by cfrikke** [reply]
Awesome! I use chrome, but I found it on chrome 😄
